src/trading/ClaimProceeds.sol

```diff
@@ -0,0 +1,50 @@
+pragma solidity ^0.4.13;
+
+import 'ROOT/Controller.sol';
+import 'ROOT/reporting/Interfaces.sol';
+import 'ROOT/extensions/MarketFeeCalculator.sol';
+
+/**
+ * @title ClaimProceeds
+ * @dev This allows users to claim their money from a market by exchanging their shares 
+ */
+contract ClaimProceeds is Controlled {
+	uint256 private constant THREE_DAYS = 3 * 24 * 60 * 60;
+
+	function publicClaimProceeds(IMarket market) onlyInGoodTimes public returns(uint256) {
+		require(market.isFinalized());
+		require(block.timestamp > market.getFinalizationTime() + THREE_DAYS);
+
+	    uint8 outcome = 0;
+	    while (outcome < market.getNumberOfOutcomes()) {
```

Comment:
Solidity has a for loop, so this should be changed from a while loop to `for (uint8 outcome = 0; outcome < market.getNumberOfOutcomes(); ++outcome)`.

It would also be good to extract `numberOfOutcomes` out into a variable so we don't have to do a cross-transaction call with every loop iteration.